src/styles/variables.css

```diff
@@ -38,6 +38,8 @@
   --temp-grey-11: rgb(136, 136, 136); /* Used for the text color of time for the 'CardActivity' items */
   --temp-grey-12: rgb(152, 164, 193); /* Used for Drag & Drop placeholder */
   --temp-grey-13: rgb(227, 224, 227); /* Used for separator background */
+  --temp-grey-14: rgb(244, 241, 244); /* Used for top nav background */
+  --temp-grey-15: rgba(60, 68, 77, 0.2); /* Used for top nav border */
```

Comment:
I am not sure that we should add the colours to the var file if they are just used once... I understand that it's a core element as a pro to do that but still not convinced.
I don't actually understand why we are doing a different border and background colours for this nav bar vs. what we already have for the actions/motions page. i see that you implemented the spec - this is a question to @arren.

Reply:
Yes, good pickup. It is best to use what is already in the DApp. Thanks @chinins!